tests/test_workers/test_tasks/test_build_merge_index_image.py

```diff
@@ -57,7 +57,7 @@ def test_handle_merge_request(
     mock_pi,
     mock_vii,
     mock_uiips,
-    mock_sir,
+    mock_sr,
```

Comment:
Trivial: mock_ors instead of mock_sr (I assume mock_sr was used with previous patch but the function changed later)